src/fhirConfig.ts

```diff
@@ -113,9 +113,19 @@ export interface Profile {
     resources?: Resources;
 }
 
-export interface FhirConfig {
+export interface ProductInfo {
     orgName: string;
+    productVersion?: string;
+    productName?: string;
+    productSlug?: string;
```

Comment:
What is this attribute?

Reply:
Terrible name choice on my part TBH, but the intent is to map this to the `CapabilityStatement.name` field which represents the machine-processing friendly name identifying the Capability Statement: 

https://www.hl7.org/fhir/capabilitystatement-definitions.html#CapabilityStatement.name

Suggestions welcomed on the naming of any/all of these.

Reply:
Hmm maybe change 
`productName`-> `productTitle`
&&
`productSlug`-> `productMachineName`

Also something to help IDEs/ other devs is you can add comments describing the fields like this:
```suggestion
/**
  * Name should be usable as an identifier for the module by machine processing applications such as code generation.
  */
    productMachineName?: string;
```

Reply:
I added comments describing each property in the `ProductInfo` interface. Everything in `ProductInfo` thus far is related to information that appears in the capability statement. Should this interface just be renamed to something along the lines of `CapabilityStatementInfo`?